examples/detection_callback.py

```diff
@@ -24,6 +25,13 @@ def simple_callback(device: BLEDevice, advertisement_data: AdvertisementData):
 
 
 async def main(service_uuids):
+    if service_uuids[0] == "all":
+        logger.info("Adding all known service_uuids")
+        service_uuids.pop(0)
+        for item in uuid16_dict:
+            service_uuids.append("{0:04x}".format(item))
+        for item in uuid128_dict:
+            service_uuids.append(item)
```

Comment:
If you like one-liners.

```suggestion
            service_uuids.extend(uuid128_dict.keys())
```

Reply:
done